ui/src/app/certificate_requests/row.tsx

```diff
@@ -67,7 +67,14 @@ export default function Row({ id, csr, certificate, ActionMenuExpanded, setActio
             <td id="expanded-row" className="p-table__expanding-panel" aria-hidden={detailsMenuOpen ? "false" : "true"}>
                 <div className="row">
                     <div className="col-8">
-                        <p><b>Common Name</b>: {csrObj.subjects.find((e) => e.type == "Common Name")?.value}</p>
+                        <p><b>Common Name</b>: {csrObj.commonName || "N/A"}</p>
+                        <p><b>Organization</b>: {csrObj.organization || "N/A"}</p>
+                        <p><b>Country</b>: {csrObj.country || "N/A"}</p>
+                        <p><b>Locality</b>: {csrObj.locality || "N/A"}</p>
+                        <p><b>Email Address</b>: {csrObj.emailAddress || "N/A"}</p>
+                        <p><b>Subject Alternative Name DNS</b>: {(csrObj.sansDns && csrObj.sansDns.length > 0) ? csrObj.sansDns.join(', ') : "N/A"}</p>
+                        <p><b>Subject Alternative Name IP addresses</b>: {(csrObj.sansIp && csrObj.sansIp.length > 0) ? csrObj.sansIp.join(', ') : "N/A"}</p>
```

Comment:
lets not have the paranthesis in the beginning of the tertiary operator to be consistent with the rest of the code

Reply:
✅